tests/pull-test.sh

```diff
@@ -114,13 +131,14 @@ ${CMD_PREFIX} ostree --repo=mirrorrepo prune --refs-only
 ${CMD_PREFIX} ostree --repo=mirrorrepo pull --bareuseronly-files origin main
 echo "ok pull (bareuseronly, safe)"
 
+if ! is_bare_user_only_repo repo; then
```

Comment:
Wait, why doesn't this work in a bare-user-only repo?

Actually, looking at all the commands in this test, none of them actually use the `repo` repo, right? I.e. it's basically independent of `repo_mode` it seems. Might be missing something though.

Reply:
Fixup ⬇️